std-bits/table/src/main/java/org/enso/table/write/ExistingDataMode.java

```diff
@@ -0,0 +1,22 @@
+package org.enso.table.write;
+
+public enum ExistingDataMode {
+  /** Throw an ExistingDataException if the target already contains data. */
+  ERROR,
+  /** The current data is replaced by the new data. */
+  REPLACE,
+  /**
+   * The new data is appended to the current data based on column position. If the new data has
+   * fewer columns than the existing data, extra columns are NULL. If the new data has more columns
+   * than the existing data, extra columns are dropped. If mismatched, a ColumnCountMismatch is
+   * attached to the result.
+   */
```

Comment:
Also, I'd suggest changing the wording slightly because the current wording seems to suggest (at least that's how I read it) that the 'mismatched' case is a separate case from the first two, but it's not - it's one of these two.
```suggestion
  /**
   * The new data is appended to the current data based on column position. If the new data has
   * fewer columns than the existing data, extra columns are NULL. If the new data has more columns
   * than the existing data, extra columns are dropped. On either mismatch, a ColumnCountMismatch is
   * attached to the result.
   */
```

Reply:
The implementation of `ColumnMapper` does not seem to align with these comments - on either mismatch it throws an error, it does not seem to try the 'recovery' of adding NULL columns for missing ones or dropping extra columns. The implementation and documentation should be aligned.